### DISCUSSION TITLE
Mirror control ptz

### DISCUSSION THREAD

**Discussion by whakru** [blakeblackshear/frigate#12237] [Ask A Question]
My ptz does not work correctly with the left and right buttons.  How to make it mirror?

**Comment by NickM-27** [comment]
usually cameras will have an option for this

**Reply by whakru** [reply]
The camera itself works well in other applications, but if I reverse it in the camera I get bad results in other applications.

**Reply by hawkeye217** [reply]
Frigate just sends a standard ONVIF movement call, so there's nothing unique about what Frigate does compared to other applications that use ONVIF. Perhaps your other apps are not using ONVIF?

**Reply by whakru** [reply]
The native camera app works great.

**Reply by whakru** [reply]
need a software mirror option...

**Reply by hawkeye217** [reply]
> The native camera app works great.

The native camera app probably doesn't use ONVIF, then.

**Reply by whakru** [reply]
> > The native camera app works great.
> 
> The native camera app probably doesn't use ONVIF, then.

quite possible.  But the problem is relevant in any case.  and I would like to solve it programmatically in frigate
